client/state/rewind/retention/reducer.ts

```diff
@@ -0,0 +1,26 @@
+import { AnyAction } from 'redux';
+import {
+	JETPACK_BACKUP_RETENTION_UPDATE,
+	JETPACK_BACKUP_RETENTION_UPDATE_ERROR,
+	JETPACK_BACKUP_RETENTION_UPDATE_SUCCESS,
+} from 'calypso/state/action-types';
+import { combineReducers } from 'calypso/state/utils';
+import { AppState } from 'calypso/types';
+
+const updateBackupRetentionRequestStatus = ( state: AppState = null, { type }: AnyAction ) => {
+	switch ( type ) {
+		case JETPACK_BACKUP_RETENTION_UPDATE:
+			return 'pending';
```

Comment:
Maybe it would be better to add the different request statuses as constants and use it anywhere, so we don't need to hard-code the strings anytime we need to validate the status. It can be something like this:

```
const BACKUP_RETENTION_UPDATE_PENDING = 'pending';
const BACKUP_RETENTION_UPDATE_SUCCESS = 'success';
const BACKUP_RETENTION_UPDATE_FAILED = 'failed';
```

What do you think?

Reply:
Good suggestion. Maybe we can declare such constants without being specific to backup retention API, given that those are generic states an API call can have and can be reused with other API calls in the future. What do you think about moving these to the root of the rewind/backup folder? like this:

```
const BACKUP_API_REQUEST_PENDING = 'pending';
const BACKUP_API_REQUEST_SUCCESS = 'success';
const BACKUP_API_REQUEST_FAILED = 'failed';
```

Reply:
Agreed with @Initsogar. We usually declare those as `constants.ts` in the state subtree directory and reuse them where needed.